LICENSE.md

```diff
@@ -1,16 +1,31 @@
+BSD 3-Clause License
+
 LICENSE
 =======
-Copyright (c) 2014, Carl Zeiss Industrielle Messtechnik GmbH
+Copyright (c) 2014, Carl ZEISS Industrielle Messtechnik GmbH
```

Comment:
In the context of the legal entity Zeiss should not be used upper case: "Carl Zeiss Industrielle Messtechnik GmbH".

Reply:
There is a document on that. I'll send you a copy.